File: maximumintable.java
import java.util.Scanner;

public class maximumintable{

	public static void main(String[]args){

		Scanner in = new Scanner(System.in);
		int n = in.nextInt();
		int [][] ans = new int[n][n];
		
		for(int row = 0; row < n; row++){
			for(int col = 0; col < n; col++){
				
				if(row == 0){
					ans[row][col] = 1;
				}
				else if(col == 0){
					ans[row][col] = 1;
				}
				else{
					ans[row][col] = ans[row - 1][col] + ans[row][col - 1];
				}
			}
		}
		System.out.println(ans[n - 1][n - 1]);
	}
}

File: maximumintable.py
n = int(input())
ans = [[x for x in range(n)] for y in range(n)]

for row in range(n):
    for col in range(n): 
        if row == 0:
            ans[row][col] = 1
        elif col == 0:
            ans[row][col] = 1
        else:
            ans[row][col] = ans[row-1][col] + ans[row][col-1]

print(ans[-1][-1])

